src/main/kotlin/ink/abb/pogo/scraper/Bot.kt

```diff
@@ -125,6 +133,16 @@ class Bot(val api: PokemonGo, val settings: Settings) {
             WebServer().start(settings.guiPort, settings.guiPortSocket)
             ctx.server.start(ctx, settings.guiPortSocket)
         }
+
+
+        if (settings.timerWalkToStartPokeStop != -1L)
```

Comment:
Here it's better if you put `settings.timerWalkToStartPokeStop<=0`.
What if someone puts -2 as timerWalkToStartPokeStop?